<div class="chatlog__message-group">
  <div class="chatlog__author-avatar-container">
    <img class="chatlog__author-avatar" src="https://cdn.discordapp.com/avatars/338302215078936576/2f0f5d2880a9a7bdc037652fbade2e9a.png?size=128" alt="Avatar" loading="lazy">
  </div>
  <div class="chatlog__messages">
    <span class="chatlog__author-name" title="hironytic#5238" data-user-id="338302215078936576" style="color: rgb(241,196,15)">hironytic</span>
    <span class="chatlog__timestamp">18-Oct-19 03:43 AM</span>
    <div class="chatlog__message " data-message-id="634597493983936518" id="message-634597493983936518" title="Message sent: 18-Oct-19 03:43 AM">
      <div class="chatlog__content">
        <div class="markdown">
          <span class="preserve-whitespace">話がそれちゃうけど、この例のFailureがNeverのときの具体的な型って、next()はthrows付きとthrowsなしの2つがオーバーロードされた感じになってるんですか？それともなしのほうだけになってる？</span>
        </div>
      </div>
    </div>
  </div>
</div>